src/domain/shared/components/Backdrops/ImageBackdrop.tsx

```diff
@@ -12,32 +12,43 @@ const Root = styled(Box)(() => ({
 }));
 
 const Background = styled(ImageFadeIn)(() => ({
+  display: 'grid',
+  placeContent: 'center',
   width: '100%',
-  height: '100%',
+
+  '@media (width > 600px)': {
```

Comment:
This hardcoded `> 600px` is not ideal. Let's be consistent and use the [theme.breakpoints.up('sm')]

Same for the other occasions.